src/dms/core/EdFi.DataManagementService.Core.Tests.Unit/Handler/DeleteByIdHandlerTests.cs

```diff
@@ -62,7 +64,7 @@ public override Task<DeleteResult> DeleteDocumentById(IDeleteRequest deleteReque
             }
         }
 
-        private readonly PipelineContext _context = No.PipelineContext();
+        private readonly RequestData _context = No.RequestData();
```

Comment:
Renamed PipelineContext to RequestData. I gave it the original name but hate the word "Context"